src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.ts

```diff
@@ -61,6 +62,8 @@ export class LogsComponent implements OnInit, OnDestroy {
   getInfo() {
     this.logsService.getLogs().subscribe((data: any) => {
       this.contentData = data;
+      this.clogText = this.logToText(this.contentData.clog);
```

Comment:
Just a suggestion: instead of cheking the length of the message, what about checking that the content is the expected one?
Something like this:
```
diff --git a/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.spec.ts b/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.spec.ts
index 862603f78a0..69c6051d2d3 100644
--- a/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.spec.ts
+++ b/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.spec.ts
@@ -144,27 +144,26 @@ describe('LogsComponent', () => {
 
   describe('convert logs to text', () => {
     it('convert cluster & audit logs to text', () => {
-      logsServiceSpy.and.returnValue(
-        of({
-          clog: [
-            {
-              name: 'priority',
-              stamp: '2019-02-21 09:39:49.572801',
-              message: 'Manager daemon localhost is now available',
-              priority: '[ERR]'
-            }
-          ],
-          audit_log: [
-            {
-              stamp: '2020-12-22T11:18:13.896920+0000',
-              priority: '[INF]'
-            }
-          ]
-        })
-      );
+      const logsPayload = {
+        clog: [
+          {
+            name: 'priority',
+            stamp: '2019-02-21 09:39:49.572801',
+            message: 'Manager daemon localhost is now available',
+            priority: '[ERR]'
+          }
+        ],
+        audit_log: [
+          {
+            stamp: '2020-12-22T11:18:13.896920+0000',
+            priority: '[INF]'
+          }
+        ]
+      };
+      logsServiceSpy.and.returnValue(of(logsPayload));
       fixture.detectChanges();
-      expect(component.clogText.length).toBe(73);
-      expect(component.auditLogText.length).toBe(42);
+      expect(component.clogText).toContain(logsPayload.clog[0].message);
+      expect(component.auditLogText).toContain(logsPayload.audit_log[0].priority);
     });
   });
 });

```